### ISSUE TITLE
Use different sans-serif font for headlines

### ISSUE THREAD

**Issue by derric** [derric/cleanthesis#132]
The plan is to use a sans-serif font, which provides more font styles/typefaces. Currently I am thinking of Roboto (regular, bold, medium, black, light, thin).

**Comment by i-am-the-slime** [comment]
Why not use a good-looking font instead of Roboto? I've had nice results with Inter.

**Comment by derric** [comment]
Yes, the Inter would also be an option - I will give it a try to see how it looks. Maybe a new option for a few sans-serif fonts for headings would be good - or not good in the sense of configuration complexity. Seems I have to think about it.
Anyway, thanks for the suggestion.
